### PR TITLE
Rotation in permalink

### PR DESCRIPTION
**Pull request by llienher**
Had to set it to all the instances of view created across the project, otherwise it would just not set the rotation and remove it to default value (0 but not the 0 provided from me at 'rotation' variable creation).

https://jira.camptocamp.com/browse/GSLUX-373

### CODE REVIEW

File: geoportal/geoportailv3_geoportal/static-ngeo/js/catalog/CatalogController.js [hunk 1]
```diff
@@ -135,15 +142,23 @@ exports.prototype.setThemeZooms = function(tree) {
...
+
     map.setView(new olView({
       maxZoom: maxZoom,
       minZoom: 8,
       extent: this.maxExtent_,
       center: currentView.getCenter(),
       enableRotation: true,
-      zoom: currentView.getZoom()
+      zoom: currentView.getZoom(),
+      rotation,
```
Comment (mki-c2c) [inline geoportal/geoportailv3_geoportal/static-ngeo/js/catalog/CatalogController.js:161]: why not like `rotation: rotation` ?
(but I must admit, I don't really know the application context, my comment is based on similarity...)

**Review by mki-c2c — APPROVED** [review]
I leave testing to @tonio and made a very superficial review.

**Review by tonio — APPROVED** [review]
Tested, work great, thanks!

Code could be simplified a bit, cf comment, please merge when it's done.

File: geoportal/geoportailv3_geoportal/static-ngeo/js/catalog/CatalogController.js (hunk 1 above)
Comment (tonio) [reply geoportal/geoportailv3_geoportal/static-ngeo/js/catalog/CatalogController.js:161]: it's es6 syntaxic sugar: when the key has the same name as the variable, you can shortcut like this.

Comment (llienher) [reply geoportal/geoportailv3_geoportal/static-ngeo/js/catalog/CatalogController.js:161]: When using ES6 syntax, you can skip the redendancy of writing key: value when it is the same to just write the key. The compiler will interpret the parameter key as the same as the value.

Comment (tonio) [reply geoportal/geoportailv3_geoportal/static-ngeo/js/catalog/CatalogController.js:161]: prem's (-;

Comment (llienher) [reply geoportal/geoportailv3_geoportal/static-ngeo/js/catalog/CatalogController.js:161]: :-(

Comment (mki-c2c) [reply geoportal/geoportailv3_geoportal/static-ngeo/js/catalog/CatalogController.js:161]: ne vous battez pas tout de meme.
thanks for the clarifications

File: geoportal/geoportailv3_geoportal/static-ngeo/js/catalog/CatalogController.js [hunk 2]
```diff
@@ -135,15 +142,23 @@ exports.prototype.setThemeZooms = function(tree) {
       var resolutions = tree['metadata']['resolutions'];
       maxZoom = resolutions.length + 7;
     }
+  
     var map = this['map'];
     var currentView = map.getView();
+
+    let rotation = 0;
+    if (this.ngeoLocation_.getParam('rotation') !== undefined) {
```
Comment (tonio) [inline geoportal/geoportailv3_geoportal/static-ngeo/js/catalog/CatalogController.js:150]: you could have used the same simplification here

**Pull request merged**